proto/stride/stakeibc/tx.proto

```diff
@@ -314,3 +316,27 @@ message MsgUpdateTradeRoute {
   ];
 }
 message MsgUpdateTradeRouteResponse {}
+
+// Registers or updates a community pool rebate by specifying the amount liquid
+// staked
+message MsgSetCommunityPoolRebate {
+  option (cosmos.msg.v1.signer) = "creator";
+  option (amino.name) = "stride/x/stakeibc/MsgSetCommunityPoolRebate";
+
+  // Message signer (admin only)
+  string creator = 1 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
+  // Chain id of the chain whose community pool has a liquid staking rebate
+  // arrangement with stride
+  string chain_id = 2;
+  // Rebate percentage (e.g. 0.2 for 20%)
+  string rebate_percentage = 3 [
```

Comment:
nit: `rebate_percentage` having 0.2 represent 20% is confusing - since it's called percentage, 0.2 could be misinterpreted as 0.2%. suggest either renaming or setting the value to 20

Reply:
What do you think of `rebate_rate`? `rebate_fraction` also seems good, either works!

Reply:
`rebate_rate` works! just updated. Will update the field on the host zone in a different PR